### DISCUSSION TITLE
[Projects Beta] Sum of numeric fields

### DISCUSSION THREAD

**Discussion by gaaamii** [community/community#15664] [Projects and Issues]
In Projects Beta, I want to view the sum of numeric fields as a row. This is to be used to observe the velocity of the sprint in Scrum development.

My sample project is [here](https://github.com/users/gaaamii/projects/1/). And I want to view the sum of "Point" fields.

**Comment by nkolev92** [comment]
I'd love this feature. 
I find myself switching between the chart and board views and that gets old quickly.

**Comment by jeffmaher** [comment]
Big +1 to this feature. Other uses cases are:

- Hours fields can be used to see if a team is committing too much or too little within a phase of work
- ^ Same for measuring story point commitment (i.e. when did we add too many stories?)
- Cost of work for a phase

**Comment by eldamir** [comment]
I have issues laid out in sprints. Each task has points. When I "Group by" the sprint, it would be natural to see common aggregates like:

- row count per sprint
- Sum of any number field per sprint (sum of points in my case)

**Comment by davidschober** [comment]
+100 This is critical if a team uses points or hours.

**Comment by dsfai2020** [comment]
I'd like this too.  I want to be able to see the total amount of points I have in each iteration.  This would be really helpful.

**Comment by easterncoder** [comment]
+1 for this feature. In my case, say I have 3 numeric fields - Difficulty, Marketability, Demand. Creating a custom field called "Priority" which sums the values of these 3 fields in a row would be awesome. Thanks!

**Comment by dwainm** [comment]
You can already do this if you have a numeric field and use the group by feature. You can only have this with grouping items. 
<img width="314" alt="Screenshot 2023-02-16 at 07 53 40" src="https://user-images.githubusercontent.com/1713474/219280272-21075851-7520-42fc-8a67-54a45dcf67ac.png">

<img width="538" alt="Screenshot 2023-02-16 at 07 55 10" src="https://user-images.githubusercontent.com/1713474/219280516-7ebe98f1-0f22-4d81-96a0-762ed0af0196.png">

**Reply by eldamir** [reply]
@dwainm we couldn't when we asked. But you're right, it is a feature now, so I suppose this can be closed 😊
